Tests/Microsoft.AppCenter.Crashes.Test.Windows/HandledErrorTest.cs

```diff
@@ -0,0 +1,237 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AppCenter.Channel;
+using Microsoft.AppCenter.Crashes.Ingestion.Models;
+using Microsoft.AppCenter.Ingestion.Http;
+using Microsoft.AppCenter.Ingestion.Models;
+using Microsoft.AppCenter.Ingestion.Models.Serialization;
+using Microsoft.AppCenter.Storage;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+namespace Microsoft.AppCenter.Crashes.Test.Windows
+{
+    [TestClass]
+    public class HandledErrorTest
+    {
+        private static IHttpNetworkAdapter StartCrashes()
```

Comment:
Can't it be `TestInitialize`? The method is called at the beginning of all tests.